rdmo/projects/assets/js/interview/components/main/question/QuestionReuseValue.js

```diff
@@ -0,0 +1,102 @@
+import React, { useState, useEffect } from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+import { isEmpty } from 'lodash'
+
+import useLsState from 'rdmo/core/assets/js/hooks/useLsState'
+
+import Modal from 'rdmo/core/assets/js/components/Modal'
+
+import Search from '../Search'
+
+const QuestionReuseValue = ({ page, question, value, updateValue }) => {
+  const initialFormValues = {
+    value: ''
+  }
+
+  const [show, setShow] = useState(false)
+  const [
+    formValues, setFormValues, initFormValues
+  ] = useLsState('rdmo.interview.reuse', initialFormValues, ['value'])
+  const [formErrors, setFormErrors] = useState([])
+
+  const handleSubmit = (mode) => {
+    if (isEmpty(formValues.value)) {
+      setFormErrors({ value: true })
+    } else {
+      const { text, option, external_id } = formValues.value
+
+      if (mode == 'append') {
+        updateValue(value, { text: `${value.text} ${text}`, option, external_id })
```

Comment:
I was trying out and thought that maybe `append` should not do anything when the same `value.text` is already there. I have here some implementation that also trims white spaces. What do you think about this?
```js
            const trimmedText = text.trim()
            const currentText = value.text?.trim() || ''

            if (currentText === trimmedText) {
                setShow(false)
                return
            }

            updateValue(value, { text: `${currentText} ${trimmedText}`.trim(), option, external_id })
```

Reply:
No, it just seemed weird that I could append the same text, let's see in user testing about it..